libfdt/fdt_ro.c

```diff
@@ -522,6 +522,18 @@ uint32_t fdt_get_phandle(const void *fdt, int nodeoffset)
 	return fdt32_ld_(php);
 }
 
+static const void *fdt_path_getprop_namelen(const void *fdt, const char *node,
+					    const char *name, int namelen,
+					    int *lenp)
```

Comment:
I think the parameter naming is not optimal. `node` isn't a `struct node` but the path name of the node to lookup `name` in. Maybe use `s/node/nodename/` and `s/name/propname/`?

Reply:
Ack, grepping clearly shows that we only have `const char *nodename` and `const char *node_name`. And `propname` is also not unheard of, and makes sense here where there's also a nodename.

Reply:
`path` would be better, matching `fdt_path_offset()`.